mgmtsystem/virtualcenter.py

```diff
@@ -422,7 +422,13 @@ def vm_status(self, vm_name):
     def vm_creation_time(self, vm_name):
         # psphere turns date strings in datetime for us
         vm = self._get_vm(vm_name)
-        return vm.runtime.bootTime
+        try:
+            return vm.runtime.bootTime
+        except AttributeError as e:
+            self.logger.error(e)
+            vm_uptime = vm.summary.quickStats.uptimeSeconds
+            vm_delta = timedelta(seconds=int(vm_uptime))
+            return datetime.now() - vm_delta
```

Comment:
Should we instead use the providers time, just in case the server running it screws up its clock?